client/my-sites/customer-home/main.jsx

```diff
@@ -0,0 +1,288 @@
+import { getCustomizerUrl, canCurrentUserUseCustomerHome } from 'state/sites/selectors';
+import isSiteEligibleForCustomerHome from 'state/selectors/is-site-eligible-for-customer-home';
+import PageViewTracker from 'lib/analytics/page-view-tracker';
+import DocumentHead from 'components/data/document-head';
+import getSiteChecklist from 'state/selectors/get-site-checklist';
+import isSiteChecklistComplete from 'state/selectors/is-site-checklist-complete';
+import QuerySiteChecklist from 'components/data/query-site-checklist';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const ActionBox = ( { href, onClick, iconSrc, label } ) => {
+	const buttonAction = { href, onClick };
+	return (
+		<div className="customer-home__box-action">
+			<Button { ...buttonAction }>
+				<img src={ iconSrc } alt="" />
+				<span>{ label }</span>
+			</Button>
+		</div>
+	);
+};
+
+class Home extends Component {
+	static propTypes = {
+		checklistMode: PropTypes.string,
+
+		site: PropTypes.object.isRequired,
+		siteId: PropTypes.number.isRequired,
+		siteSlug: PropTypes.string.isRequired,
+		customizeUrl: PropTypes.string.isRequired,
+		canUserUseCustomerHome: PropTypes.bool.isRequired,
+		hasChecklistData: PropTypes.bool.isRequired,
+		isChecklistComplete: function( props, propName, componentName ) {
+			const propValue = props[ propName ]; // the actual value of `isChecklistComplete`
+			if ( null !== propValue && 'boolean' !== typeof propValue ) {
+				return new Error(
+					`isChecklistComplete prop of ${ componentName } only accepts null or Boolean.`
+				);
+			}
+		},
+		isSiteEligible: PropTypes.bool.isRequired,
+	};
+
+	state = {
+		renderChecklistCompleteBanner: null,
+	};
+
+	static getDerivedStateFromProps( nextProps, prevState ) {
+		// If we still don't have checklist data or we don't want to render the banner, probably because
+		// this is a page load, don't change the state.
+		if (
+			null === nextProps.isChecklistComplete ||
+			'norender' === prevState.renderChecklistCompleteBanner
+		) {
+			return null;
+		}
+		// If this state prop doesn't have a value yet
+		if ( null === prevState.renderChecklistCompleteBanner ) {
+			return {
+				// Set to norender because this is the initial state value and the checklist is completed.
+				// Otherwise the banner will always display once the checklist is completed, even on page load.
+				renderChecklistCompleteBanner: nextProps.isChecklistComplete ? 'norender' : 'waiting', // If checklist is not complete, let's flag it and wait until it is.
+			};
+		}
+		// If we're here, this is not a page load, so let's check if the checklist was completed.
+		if ( nextProps.isChecklistComplete ) {
+			return {
+				renderChecklistCompleteBanner: 'render',
+			};
+		}
+	}
+
+	render() {
+		const { translate, canUserUseCustomerHome } = this.props;
+
+		if ( ! canUserUseCustomerHome ) {
+			const title = this.props.isSiteEligible
+				? translate( 'You are not authorized to view this page.' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **28** times:
`translate( 'You are not authorized to view this page' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).